viewer/packages/camera-manager/src/DefaultCameraManager.ts

```diff
@@ -545,6 +545,8 @@ export class DefaultCameraManager implements CameraManager {
     };
 
     const onWheel = async (e: any) => {
```

Comment:
Just noticed this `any`. That should be changed asap - there is no need for it....

Reply:
Can't be easily resolved because of `clickOrTouchEventOffset` : https://github.com/cognitedata/reveal/pull/1307 .
If you can explain why we need it, I can try to remove it and use normal `MouseEvent`.

Reply:
based on what I read the type will simply be `ev: MouseEvent | TouchEvent`

Reply:
It's not that simple unfortunately. `InputHandler` fires "event" with data after applying `clickOrTouchEventOffset` to `MouseEvent`. I can try to create a mock event with new data for proper typing since all fields on normal events are read only.

Reply:
Sorry, seems that I was looking at something else, works fine with a few small changes...